src/api/status/public/index.html

```diff
@@ -190,139 +236,6 @@ <h6 class="ps-4 ms-2 text-uppercase text-xs text-white font-weight-bolder opacit
             </ol>
             <h6 class="font-weight-bolder mb-0">Dashboard</h6>
           </nav>
-          <div class="collapse navbar-collapse mt-sm-0 mt-2 me-md-0 me-sm-4" id="navbar">
-            <div class="ms-md-auto pe-md-3 d-flex align-items-center">
-              <div class="input-group input-group-outline">
-                <label class="form-label">Type here...</label>
-                <input type="text" class="form-control" />
-              </div>
-            </div>
-            <ul class="navbar-nav justify-content-end">
-              <li class="nav-item d-flex align-items-center">
-                <a href="javascript:;" class="nav-link text-body font-weight-bold px-0">
-                  <i class="fa fa-user me-sm-1"></i>
-                  <span class="d-sm-inline d-none">Sign In</span>
-                </a>
-              </li>
-              <li class="nav-item d-xl-none ps-3 d-flex align-items-center">
```

Comment:
Cannot remove this side-nav-toggler. This is used for opening the sidebar on small screens.

Reply:
@Andrewnt219 Hi responsive is not in this issue.... Can you please add an issue?

Reply:
The current dashboard has this feature, you should not remove it.

![image](https://user-images.githubusercontent.com/52666982/142703771-9df2d24b-3c18-4a40-b708-e119ffccc740.png)

Reply:
fixed